controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -0,0 +1,90 @@
+import sys
+import contextlib
+from typing import Iterator
+from pathlib import Path
+
+# Webots specific library
+from controller import Supervisor  # isort:skip
+
+REPO_ROOT = Path(__file__).resolve().parent.parent.parent
+
+sys.path.append(str(REPO_ROOT / 'controllers/sr_controller'))
+
+import sr_controller  # noqa:E402 # isort:skip
+
+TIME_STEP = 32
```

Comment:
I'd be interested in trying to reuse the value in `sr.robot.settings` here if possible.

Reply:
The amount of import path munging needed to achieve that is quite high. 232481c4ceb59888eb2f44f01925aebdb917877a cross links the two instead.